data/human/human missions.txt

```diff
@@ -2502,6 +2502,123 @@ event "terraforming follow-up"
 
 
 
+mission "Terraforming 13"
+	minor
+	name "Field Trip"
+	description "Amy, who you helped terraform Rand and Tundra, seems to share some of her terraforming philosophy with the Wanderers. Bring her to <destination> to visit them."
+	source "Tundra"
+	destination "Desi Seledrak"
+	passengers 1
+	to offer
+		has "event: terraforming follow-up"
+		has "wanderers sestor done"
+	on offer
+		conversation
+			`As you walk through the spaceport, you notice a poster featuring Amy. You previously helped her terraform this world and Rand. The poster describes that terraforming action, and the long-term goal to restore the warmer climate Tundra had millions of years ago. Amy's work on "equilibrium mapping," the idea that planets naturally "want" to support more complex and varied life, seems similar to the Wanderer philosophy that each planet has its own "will", and that the best way to terraform it is to work with how that "will." Amy may be interested in learning about the Wanderers and their techniques. Would you like to find her and tell her about them?`
+			choice
+				`	(Yes.)`
+				`	(No.)`
+					decline
+			`	You're able to contact Amy. She has been continuing to monitor the status of Tundra's climate, alongside the researchers from the Deep who she worked with to terraform this world. You tell her you've encountered some alien terraformers who may share her philosophy. She is eager to learn more about them and agrees to meet you in the spaceport bar.`
```

Comment:
The first sentence is a bit of a tautology. It doesn't sound too great to me.
"Surprisingly, you're able to contact Amy much quicker than you expected."
Being able to contact her unexpectedly quickly itself implies surprise, since it was unexpected.

But, also, I didn't have any particular expectation on how long it'd take to contact her. Are we trying to make a point that she was exceptionally easy to contact? If so, why?

"a layman summation of your travels since then and briefly inform her" seems kind of verbose, and I don't think it even says what I want it to. Especially in conjunction with the next sentence beginning with: "This catches her attention."
This is the thing I came here to tell her about. It makes it sound like our character is making some long presentation and only when this one thing comes up does she actually seem interested.

Reply:
For point one, I don't have anything against it, and I leave it up to your judgement. 

For the next point, the goal here was to try and link both sentences together better. In the aforementioned text: "You tell her you've encountered some alien terraformers who may share her philosophy. She is eager to learn more about them-" This tells me that, yes, the player tells her about the Wanderer and that she wants to know more. However, the way it's presented feels too barebones. It's a near "action, reaction" sort of text. 

Why I wanted "layman summation" was more to at least give the player some sense that there was some context given, however long the player believes the conversation went, rather than "Hello, there's aliens who've their views on terraforming." I also wanted the text to be more fleshed out, to feel a bit more human, to keep the engagement from the reader.

Perhaps we could go with "quick summary of your adventures?" or something close to that to imply a swift conversation?